docs/migrations/v4-to-v5.md

```diff
@@ -0,0 +1,107 @@
+# Migrating from v4 to v5
+
+This document is intended to highlight significant changes which may require more information than presented in the CHANGELOG.
+Any changes that must be done by a user of ibc-go should be documented here.
+
+There are four sections based on the four potential user groups of this document:
+- Chains
+- IBC Apps
+- Relayers
+- IBC Light Clients
+
+**Note:** ibc-go supports golang semantic versioning and therefore all imports must be updated to bump the version number on major releases.
+```go
+github.com/cosmos/ibc-go/v4 -> github.com/cosmos/ibc-go/v5
+```
+
+No genesis or in-place migrations required when upgrading from v1 or v2 of ibc-go.
```

Comment:
Do we still need this line here?

Reply:
i agree it could be removed, it's not very applicable to this file